fdbclient/BackupContainerFileSystem.actor.cpp

```diff
@@ -2256,7 +2256,11 @@ void printFileList(BackupFileList& backupFileList) {
 
 	printf("\nSnapshotFiles count:%lu", backupFileList.snapshots.size());
 	for (auto s : backupFileList.snapshots)
-		printf("\n%lld, %lld, %s, %lld\n", s.beginVersion, s.endVersion, s.fileName.c_str(), s.totalSize);
+		printf("\n%" PRId64 ", %" PRId64 ", %s, %" PRId64 "\n",
```

Comment:
Personally I prefer cout over printf since cout parses the input during the compile time.